.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### What's in there?
+
+Some context
+
+Are included here:
+
+- [ ] feature A
+- [ ] feature B
+
+### How to test?
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/gh/feugy/melodie/pullRequest?prid=6357597)